builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -326,6 +326,18 @@ def loadServiceConfigurationData() {
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/dev-config.json"
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/stg-config.json"
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/prod-config.json"
+
+            sh "sed -i -- 's/{default_provider}/${config_loader.AWS.DEFAULTS.PROVIDER}/g' ./config/dev-config.json"
+            sh "sed -i -- 's/{default_provider}/${config_loader.AWS.DEFAULTS.PROVIDER}/g' ./config/stg-config.json"
+            sh "sed -i -- 's/{default_provider}/${config_loader.AWS.DEFAULTS.PROVIDER}/g' ./config/prod-config.json"
+
+            sh "sed -i -- 's/{default_accountId}/${config_loader.AWS.DEFAULTS.ACCOUNT_ID}/g' ./config/dev-config.json"
```

Comment:
I believe we are using `ACCOUNTID` instead of `ACCOUNT_ID` as per changes in https://github.com/tmobile/jazz-installer/pull/389